### PR TITLE
Fixes in the URLs

### PR DESCRIPTION
**Pull request by eugmandel**

### CODE REVIEW

File: great_expectations/cli/datasource.py [hunk 1]
```diff
@@ -3,15 +3,16 @@
...
 
 
 def add_datasource(context):
     cli_message(
         """
 ========== Datasources ==========
 
-See <blue>https://great-expectations.readthedocs.io/en/latest/core_concepts/datasource.html?src=cli</blue> for more information about datasources.
-"""
+See <blue>http://docs.greatexpectations.io/en/latest/core_concepts/datasource.html?utm_source=cli&utm_medium=init&utm_campaign={0:s}</blue> for more information about datasources.
```
Comment (jcampbell) [inline great_expectations/cli/datasource.py]: should still be https here (vs http)

File: great_expectations/cli/datasource.py [hunk 2]
```diff
@@ -122,14 +123,14 @@ def add_datasource(context):
...
-Would you like to profile '%s' to create candidate expectations and documentation?
+Would you like to profile '{0:s}' to create candidate expectations and documentation?
 
 Please note: As of v0.7.0, profiling is still a beta feature in Great Expectations.  
 This generation of profilers will evaluate the entire data source (without sampling) and may be very time consuming. 
 As a rule of thumb, we recommend starting with data smaller than 100MB.
 
-To learn more about profiling, visit <blue>https://great-expectations.readthedocs.io/en/latest/guides/profiling.html?src=cli</blue>.
-            """ % (data_source_name)
+To learn more about profiling, visit <blue>http://docs.greatexpectations.io/en/latest/guides/profiling.html?utm_source=cli&utm_medium=init&utm_campaign={1:s}</blue>.
```
Comment (jcampbell) [inline great_expectations/cli/datasource.py]: ditto scheme update

File: great_expectations/cli/datasource.py [hunk 3]
```diff
@@ -157,8 +158,8 @@ def add_datasource(context):
 
 Before committing, please make sure that this data does not contain sensitive information!
 
-To learn more: <blue>https://great-expectations.readthedocs.io/en/latest/guides/data_documentation?src=cli</blue>
-"""
+To learn more: <blue>http://docs.greatexpectations.io/en/latest/guides/data_documentation.html?utm_source=cli&utm_medium=init&utm_campaign={0:s}</blue>
```
Comment (jcampbell) [inline great_expectations/cli/datasource.py]: ditto

File: great_expectations/cli/init.py [hunk 4]
```diff
@@ -52,8 +52,8 @@ def scaffold_directories_and_notebooks(base_dir):
 
 If you're new to Great Expectations, this tutorial is a good place to start:
 
-    <blue>https://great-expectations.readthedocs.io/en/latest/getting_started.html?src=cli</blue>
-"""
+    <blue>http://docs.greatexpectations.io/en/latest/getting_started.html?utm_source=cli&utm_medium=init&utm_campaign={0:s}</blue>
```
Comment (jcampbell) [inline great_expectations/cli/init.py]: ditto

**Review by jcampbell — CHANGES_REQUESTED** [review]
change scheme

File: great_expectations/cli/datasource.py (hunk 1 above)
Comment (eugmandel) [reply great_expectations/cli/datasource.py]: done

**Review by jcampbell — APPROVED** [review]

**Pull request merged**